### PR TITLE
chore: update fluvio to 0.10.8

### PR DESCRIPTION
**Pull request by nacardin**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,16 +15,16 @@ log = "^0.4.17"
 
 [dependencies]
 cpython = { version = "0.7", features = ["extension-module"] }
-fluvio = { version = "0.18" }
-fluvio-future = { version = "0.4.2", features = ["task", "io", "native2_tls", "subscriber"] }
+fluvio = { features = ["admin"], git = "https://github.com/infinyon/fluvio.git", tag = "v0.10.8" }
+fluvio-future = { version = "0.5.1", features = ["task", "io", "native2_tls", "subscriber"] }
 
 thiserror = "1.0.21"
 fluvio-types = "0.4.0"
```
Comment (morenol) [inline Cargo.toml:22]: @nacardin I think that we should update this also to fetch from git

**Review by morenol — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (nacardin) [reply Cargo.toml:22]: update which? `fluvio-future`?

Comment (morenol) [reply Cargo.toml:22]: `fluvio-types`

**Review by morenol — APPROVED** [review]

**Comment by morenol** [comment]
bors retry

**Pull request merged**